helm/rook-operator/values.yaml

```diff
@@ -400,6 +400,14 @@ networkPolicy:
 # shared values with subchart.
 #
 global:
+  # namespace the Ceph components will be deployed into. Must match the
+  # namespace configured in the CephCluster CR.
+  cephClusterNamespace: "rook-ceph"
+
+  # set the ceph version to deploy. Currently only affects the toolbox
+  # pod in the ceph-cluster subchart.
+  cephVersion: ""
```

Comment:
This should be set in giantswarm/config and it should match the ceph version in the `CephCluster` CR when the cluster is deployed.